README.md

```diff
@@ -1,2 +1,65 @@
-# ros_qualisys
-ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package
+ros_qualisys
+============
+
+This is ROS wrapper around the https://github.com/qualisys/qualisys_cpp_sdk package.
+
+## Getting started
+
+### Dependencies
+
+All dependencies are listed in the [package.xml](https://github.com/Gepetto/ros-qualisys/blob/main/package.xml) file.
+
+For building the package:
+- git
+- doxygen
+- cmake
+
+For executing the package:
+- catkin + roscpp (if with ROS1)
+- ament_cmake + rclcpp (if with ROS2)
+- Python 2 or 3 (Standard Ubuntu 18.04 or higher)
+- boost (Standard Ubuntu 18.04 or higher)
+- qualisys_cpp_sdk (installable via the robotpkg ppa)
```

Comment:
`ppa` is a term reserved for stuff hosted by launchpad, while robotpkg is hosted by a third party, the LAAS-CNRS. 
```suggestion
- qualisys_cpp_sdk (installable via [robotpkg](http://robotpkg.openrobots.org/))
```